README.md

```diff
@@ -197,10 +197,13 @@ This section provides a step-by-step procedure for deploying the resource aggreg
       ```
    c. Use the following command to generate Kafka TLS certificate:
       ```
-       $ ./generate_kafka_tls.sh kafka
+       $ ./generate_kafka_certs.sh kafka
       ```
-      
-   d. Use the following command to copy the resource aggregator, the GRF  plugin and the Kafka TLS certificates:
+   d. Use the following command to generate Zookeeper TLS certificate:
+      ```
+       $ ./generate_zookeeper_certs.sh zookeeper
+      ```
+   e. Use the following command to copy the resource aggregator, the GRF  plugin and the Kafka TLS certificates:
```

Comment:
Add "zookeeper" :
Use the following command to copy the resource aggregator, the GRF  plugin and the Kafka  **& Zookeeper** TLS certificates:

Reply:
Done